www/class/centreon-clapi/centreonHost.class.php

```diff
@@ -83,6 +83,7 @@ class CentreonHost extends CentreonObject
     const ORDER_HOSTGROUP = 5;
     const MISSING_INSTANCE = "Instance name is mandatory";
     const UNKNOWN_NOTIFICATION_OPTIONS = "Invalid notifications options";
+    const UNKNOWN_GEO_COORDS = "Invalid geo coords";
```

Comment:
your constante name does not meet the value
maybe INVALID_GEO_COORDS is more consistent